endpoints/books/controller.py

```diff
@@ -24,38 +28,40 @@
 
 
 class UserBookResource(Resource):
+    @jwt_required
     def post(self):
         try:
+            current_user = get_jwt_identity()
             book = request.get_json()
+            book['user_id'] = current_user
             db.session.add(UserBook(**book))
```

Comment:
Use a handler/repository for this. Don't make database calls directly in your views/controllers. That's the job of the handler to call the repository.